### PR TITLE
add credits page for FFL

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

File: unicorecmsffl/templates/flatpage.pt [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <div metal:fill-slot="content">
 
         <div class="page section_news object-detail">
-            <div class="h1">
+            <div class="h2">
```
Comment (smn) [inline unicorecmsffl/templates/flatpage.pt:5]: Was this an unrelated bug?

**Comment by smn** [comment]
I think a test would be wise just so we don't forget about this addition.

File: unicorecmsffl/templates/flatpage.pt (hunk 1 above)
Comment (miltontony) [inline unicorecmsffl/templates/flatpage.pt:5]: Yeah, an `h2` looks better

**Comment by miltontony** [comment]
@smn test added

**Comment by smn** [comment]
:+1:

**Pull request merged**